docs/review-club/your-first-meeting.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: ReviewClubLayout
+title: Attending your first PR Review Club
+---
+
+## Attending your first PR Review Club
+
+If you’re thinking about attending PR Review Club, welcome! We love new
+participants. Below is information to help you get started as a first-timer.
+
+The club meets on {{ $meetingDay }}s at {{ $meetingTime }} in
+<a :href="$meetingPlace">Libera.chat</a>. Every other week, a BDK
```

Comment:
I think we should meet on our Discord rather than on IRC :)

Reply:
Alright, I will update with discord